components/editor/panels/package/package-content.mjs

```diff
@@ -84,6 +84,10 @@ function show_package_content({folders, files, ftd_root}) {
     let context_menu_path = new ftd2.FastnTik("", ftd_root, CONTEXT_MENU_PATH).get();
     let modified_files = new ftd2.FastnTik([], ftd_root, MODIFIED_FILES).get();
     let only_modified_files = new ftd2.FastnTik(false, ftd_root, ONLY_MODIFIED_FILES).get();
+
+    // saves open file state on every render
+    window.dispatchEvent(new CustomEvent("ide-event", {detail: {name: "save-file-tree", data: folder.get()}}));
```

Comment:
Ideally we should do this when the state changes. Also you are storing the entire folder. Store only the list of folders that are open. 

Also I was not thinking we should store it in OPFS as then it will be synced across multiple tabs. It should be stored in [session store](https://developer.mozilla.org/en-US/docs/Web/API/Window/sessionStorage).